src/benchmark.cc

```diff
@@ -145,6 +145,7 @@ class ThreadManager {
     double real_time_used = 0;
     double cpu_time_used = 0;
     double manual_time_used = 0;
+    int64_t iterations = 0;
```

Comment:
This seems like the wrong type, since we usually use `size_t` when dealing with iterations. It appears that the use of `int64_t` below is probably wrong too, but no need to address that in this patch.